.github/workflows/gradle.yml

```diff
@@ -105,6 +105,7 @@ jobs:
           org.gradle.vfs.watch=false
           EOF
 
+      # TODO
```

Comment:
Since this inverts the default connector project inclusion, I'm not sure yet where I need to add `INCLUDE_CONNECTORS=true` in CI jobs to make sure integrations are still getting built where they're supposed to be built